src/lib.rs

```diff
@@ -612,12 +623,18 @@ mod tests {
         let res = b.gread_with::<&str>(offset, StrCtx::Length(3));
         assert!(res.is_err());
         *offset = 0;
-        let astring: [u8; 3] = [0x45, 042, 0x44];
+        // FIXME: 0x2A used to be 042 -- a weird way to declare a number, probably a bug
```

Comment:
this was definitely a typo/bug :) should have been 0x42

Reply:
thx, fixed